### PR TITLE
Add License file

### PR DESCRIPTION
**Pull request by bureddy**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Copyright (c) 2014-2015      UT-Battelle, LLC. All rights reserved.
+Copyright (C) 2014-2020      Mellanox Technologies Ltd. All rights reserved.
```
Comment (mike-dubman) [inline LICENSE:2]: lets keep only this copyright, the rest can remove

**Comment by bureddy** [comment]
@mike-dubman fixed. Please review. Also kept NVIDIA copyright as NVIDIA NCCL team also had patches earlier.

**Review by mike-dubman — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Copyright (c) 2018-2020      NVIDIA CORPORATION. All rights reserved.
```
Comment (sashakot) [inline LICENSE:2]: As far I know, we don't put nvidia copyright and still use Mellanox only.

Comment (bureddy) [reply LICENSE:2]: @sashakot there are some changes in the source  from NVIDIA team before both companies are merged.

**Comment by mellanox-github** [comment]
Mellanox CI FAILed.

**Comment by mellanox-github** [comment]
Mellanox CI FAILed.

**Pull request merged**